pkgs/development/tools/kustomize/default.nix

```
@@ -15,13 +15,13 @@ buildGoModule rec {
     owner = "kubernetes-sigs";
     repo = pname;
     rev = "kustomize/v${version}";
-    sha256 = "sha256-7Ode+ONgWJRNSbIpvIjhuT+oVvZgJfByFqS/iSUhcXw=";
+    sha256 = "sha256-0pykj3vgnxjaya9krq5l9wvxrfazgcxq0la7dg60wqnbyksl2my1";
```

While we're at it we could change them to the new convention

```suggestion
    hash = "sha256-0pykj3vgnxjaya9krq5l9wvxrfazgcxq0la7dg60wqnbyksl2my1";
```